example/requirements-test.txt

```diff
@@ -1,5 +1,6 @@
-seaworthy >= 0.1.1
+seaworthy >= 0.1.2
 iso8601
 pytest >= 3.1
 pytest-catchlog
+requests
```

Comment:
Why was this added? I don't see it used anywhere?

Reply:
That was accidental, I'll remove it. (It must have wandered in while I was looking through the revision history for the seaworthy git link.)